taxi_service/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for taxi_service project.
+
+Generated by 'django-admin startproject' using Django 5.1.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+from django.conf.global_settings import AUTH_USER_MODEL

```

Comment:
The import of `AUTH_USER_MODEL` from `django.conf.global_settings` is unnecessary as it is not used in this file. You can safely remove this import.